contrib/debian/rules

```diff
@@ -289,7 +294,7 @@ override_dh_fixperms:
 	chmod 4750 $(TOP)/usr/libexec/netdata/plugins.d/local-listeners
 
 	# network-viewer
-	chmod 4750 $(TOP)/usr/libexec/netdata/plugins.d/network-viewer.plugin
+	chmod 4750 $(TOP)-plugin-network-viewer/usr/libexec/netdata/plugins.d/network-viewer.plugin
```

Comment:
Logic conflict with 

https://github.com/netdata/netdata/pull/16949/files#diff-86bc26be4b028856bd61e7517129fc0a2a0c5ffe14e57d5f18ac3a46bf7342a9R5-R12

You end up enforcing suid. No matter how we are trying to apply PoLP with capabilities. Just noticed that we do the same mistake with `systemd-journal-plugin`

Reply:
No, there is no conflict.

Nothing in the rules file gets run on end-user systems, this is run _when building the package_. The postinstall script then _only_ gets run on end-user systems.

So this is setting the SUID bit for the file in the package itself (which we _should_ be doing for anything that uses caps or SUID), and not doing anything regarding what actually gets installed.